sdk/src/main/java/io/radar/sdk/RadarApiClient.kt

```diff
@@ -74,7 +85,27 @@ internal class RadarApiClient(
         )
     }
 
-    internal fun getConfig() {
+    private fun parseMeta(res: JSONObject?): RadarTrackingOptions? {
```

Comment:
we re-use this for GET /config and POST /track, so I factored this out. 

note that the `config` field is a side-effect. it's not actually used right now, so I didn't include it in the return signature